samples/msal-node-samples/b2c-auth-code-pkce/policies.js

```diff
@@ -4,21 +4,10 @@
  * To learn more about custom policies, visit: https://docs.microsoft.com/en-us/azure/active-directory-b2c/custom-policy-overview
  */
 const b2cPolicies = {
-    names: {
-        signUpSignIn: "B2C_1_susi",
-        resetPassword: "B2C_1_reset",
-        editProfile: "B2C_1_edit_profile"
-    },
     authorities: {
         signUpSignIn: {
-            authority: "https://fabrikamb2c.b2clogin.com/fabrikamb2c.onmicrosoft.com/B2C_1_susi",
-        },
-        resetPassword: {
-            authority: "https://fabrikamb2c.b2clogin.com/fabrikamb2c.onmicrosoft.com/B2C_1_reset",
+            authority: "https://fabrikamb2c.b2clogin.com/fabrikamb2c.onmicrosoft.com/B2C_1_susi_reset_v2",
         },
-        editProfile: {
```

Comment:
Why is `edit_profile` removed?

Reply:
Wanted to just simplify **b2c-auth-code-pkce** (wouldn't mind reverting) and **b2c-silent-flow** (which actually has an issue [because of](https://github.com/Azure-Samples/ms-identity-b2c-javascript-spa/blob/main/App/authPopup.js#L26)).

**b2c-auth-code** still has edit profile and standalone password reset